src/OpenLoco/Audio/Audio.cpp

```diff
@@ -1196,32 +1142,43 @@ namespace OpenLoco::Audio
     // 0x0048AC66
     void playTitleScreenMusic()
     {
+        auto* channel = getChannel(ChannelId::title);
+        if (channel == nullptr)
+        {
+            return;
+        }
         if (isTitleMode() && _audioInitialised && _audioIsEnabled && Config::getNew().audio.playTitleMusic)
         {
-            if (!isChannelPlaying(ChannelId::title))
+            if (!channel->isPlaying())
             {
-                auto path = Environment::getPath(PathId::css5);
-                if (loadChannel(ChannelId::title, path, 0))
+                auto musicSample = loadMusicSample(PathId::css5);
+                if (channel->load(*musicSample))
                 {
-                    playChannel(ChannelId::title, 1, -500, 0, 0);
+                    channel->setVolume(-500);
```

Comment:
having all these volumes it really hard to deal with IMO, is there a way we can write a function to convert a percentage into these game values? is 0% = mute, 100% = max volume

Reply:
It's in terms of hundredth decibels. So this is -5dB. We could add a user defined literal for the conversion.

Reply:
Even so, how 'loud' is -5db? If 0db is the threshold of human hearing, then setting something to -5db means silence. If 0db is something other than human hearing threshold, that just makes these values even less intuitive.

Maybe this is just me, but I think representing volume in db in a game is not intuitive at all, except perhaps at the very final mixing stages (which we aren't at here); a percentage makes much more sense to me.

Maybe I'll look at implementing something like that after all the audio stuff is reverse-engineered/working with openal

Reply:
0dB is no modification to source volume. -100dB is silence. OpenAl works in terms of a percentage so the volume is scaled to 0 -> 1.0. dB is what you would normally use in sound engineering as its a logarithmic scale which translates better to human ears perception of sound. It would be hard to change any of this as objects define things in terms of dB so you are then asking to modify original objects or scale them on load which is post 100% implementation.

The previous PR implemented the last of the audio code. There is no more only callers into the code left to do.

Reply:
Yeah I get that, it's just that in most game engines and DAWs you set volumes on sounds effects, music, channels etc as a percentage (normally a float [0,1] in games). You only work in db in the final mix and master stage in the audio engine itself, but this code is just setting the channels volume; there is no reason to express that in db.

In any case I won't change anything for now.